.github/actions/build_deb.sh

```diff
@@ -2,33 +2,67 @@
 
 echo "Building the SIL Kit packages"
 
-orig_tarball=${SILKIT_SOURCE_URL}/${SILKIT_VERSION}-${SILKIT_VERSION_SUFFIX}/libsilkit_${SILKIT_VERSION}.orig.tar.xz
-wget --no-proxy --no-check-certificate $orig_tarball
-ret_val=$?
-if [ "$ret_val" != '0' ] ; then
-    echo "SILKIT-PKG: Could not get orig tarball from $orig_tarball"
-    exit 64
+if [ -z $SILKIT_PKG_URL ] ; then
+    SILKIT_PKG_URL="."
 fi
+debian_path="${SILKIT_PKG_URL}/debian"
 
-mkdir -p libsilkit-${SILKIT_VERSION}
-tar -xvf libsilkit_${SILKIT_VERSION}.orig.tar.xz -C libsilkit-${SILKIT_VERSION}
-ret_val=$?
-if [ "$ret_val" != '0' ] ; then
-    echo "SILKIT-PKG: Could not untar orig tarball $orig_tarball"
-    exit 64
+silkit_pkg_found=false
+# Check if the provided path has a debian directory
+if [ -d ${debian_path} ] ; then
+    silkit_pkg_found=true
 fi
 
-git -c http.sslVerify=false clone ${SILKIT_PACKAGING_REPO} -b ${SILKIT_PACKAGING_REPO_BRANCH}
-ret_val=$?
-if [ "$ret_val" != '0' ] ; then
-    echo "SILKIT-PKG: Could not clone $SILKIT_PACKAGING_REPO : $SILKIT_PACKAGING_REPO_BRANCH"
-    exit 64
+if ! $silkit_pkg_found ; then
+    echo "SILKIT-PKG: Trying git sil-kit-pkg"
+    if git ls-remote -hq ${SILKIT_PKG_URL} ; then
+        git -c http.sslVerify=false clone --depth=1 ${SILKIT_PKG_URL} -b ${SILKIT_PKG_REF} sil-kit-pkg
```

Comment:
This might be an artifact of doing it in a private repo. But it didn't want to clone from the container. I can try it without it if we are running on public runners.